src/lib/y2storage/storage_class_wrapper.rb

```diff
@@ -127,6 +127,8 @@ def wrap_class(storage_class, downcast_to: [])
       # @param method [Symbol] name of the method to define
       # @param as [String] name of the class to convert the result, nil if the
       #   value must be returned as-is (after turning any vector into an array)
+      # @param check_with [Symbol] name of a check method. This method is used
```

Comment:
Too much explanation of WHY `check_method` exists and too little about WHAT it means.